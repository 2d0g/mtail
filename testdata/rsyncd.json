[
  {
    "Name": "bytes_total",
    "Kind": 0,
    "D": null,
    "Keys": [
      "operation"
    ],
    "Values": {
      "received": {
        "Value": 975017,
        "Time": "2011-02-23T05:54:10Z"
      },
      "sent": {
        "Value": 62793673,
        "Time": "2011-02-23T05:54:10Z"
      }
    }
  },
  {
    "Name": "connections_total",
    "Kind": 0,
    "D": {
      "Value": 52,
      "Time": "2011-02-22T21:54:13Z"
    },
    "Keys": null,
    "Values": null
  },
  {
    "Name": "connection-time_total",
    "Kind": 0,
    "D": {
      "Value": 1181011,
      "Time": "2011-02-23T05:54:10Z"
    },
    "Keys": null,
    "Values": null
  },
  {
    "Name": "transfers_total",
    "Kind": 0,
    "D": null,
    "Keys": [
      "operation",
      "module"
    ],
    "Values": {
      "send module": {
        "Value": 2,
        "Time": "2011-02-23T05:50:32Z"
      },
      "send repo": {
        "Value": 25,
        "Time": "2011-02-23T05:51:14Z"
      }
    }
  }
]
