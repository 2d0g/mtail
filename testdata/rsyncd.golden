counter bytes_total {operation=sent} 62793673 2011-02-23T05:54:10Z
counter bytes_total {operation=received} 975017 2011-02-23T05:54:10Z
counter connections_total 52 2011-02-22T21:54:13Z
